mlog/log.go

```diff
@@ -51,6 +52,20 @@ type Logger struct {
 	fileLevel    zap.AtomicLevel
 }
 
+// LogWriter is an io.Writer that writes through t.Log
```

Comment:
Let's move this into a `testing.go` file in the same package, and rename as `TestingWriter`. Also, does the struct need to be exported or can it stay private?

Reply:
Should I move all the functions regarding the testing logger to the said file as well?

Reply:
Moved everything related to `testingLogger` to a file named testing.go under the package mlog